root/static/scripts/common/MB/Control/Autocomplete.js

```diff
@@ -487,19 +488,32 @@ $.widget('mb.entitylookup', $.ui.autocomplete, {
       }
     }
 
-    if (userCanAdd && addNewEntityLabels[entity]) {
-      const label = addNewEntityLabels[entity]();
+    if (userCanAdd && ADD_NEW_ENTITY_TITLES[entity]) {
+      const label = ADD_NEW_ENTITY_TITLES[entity]();
       results.push({
         label,
         action: function () {
-          $('<div>').appendTo('body').createEntityDialog({
-            name: self._value(),
-            entity: entity,
-            title: label,
-            callback: function (item) {
-              self.options.select(null, { item: item });
-            },
-          });
+          const containerNode = self._getAddEntityContainer()[0];
+
+          const closeAndReturnFocus = () => {
+            ReactDOM.unmountComponentAtNode(containerNode);
+            self.element.focus();
+          };
+
+          /* eslint-disable react/jsx-no-bind */
```

Comment:
You have mentioned wanting to disable this rule globally before, FWIW.

Reply:
Well, it's not that bad since we set `ignoreDOMComponents: true` on it. (In fact this is the only place I can see that we disable it anymore.)